### PR TITLE
index pages and user stuff

### PR DESCRIPTION
**Pull request by aizon2456**

### CODE REVIEW

File: app/views/pages/_header.html.erb [hunk 1]
```diff
@@ -20,16 +20,25 @@
...
           <div class="ui dropdown item">
-            Profile
+            <%= current_user.username %>
             <i class="dropdown icon"></i>
             <div class="menu">
-              <%= link_to current_user.username, current_user, class: 'item' %>
+              <%= link_to "Profile", current_user, class: 'item' %>
               <%= link_to "New Team", new_team_path, class: 'item' %>
-              <%= link_to 'Settings', edit_user_path(current_user), class: 'item' %>
+              <%= link_to "Settings", edit_user_path(current_user), class: 'item' %>
```
Comment (Rosengren) [inline app/views/pages/_header.html.erb:41]: Didn't like my single quotes? :)

File: db/schema.rb [hunk 2]
```diff
@@ -48,7 +48,6 @@
     t.datetime "date"
     t.text     "content"
     t.integer  "proficiency_id"
-    t.string   "slug"
```
Comment (Rosengren) [inline db/schema.rb:51]: Proficiencies don't need a slug?

**Comment by Rosengren** [comment]
Looks good! Check to make sure the schema is right

File: app/views/pages/_header.html.erb (hunk 1 above)
Comment (aizon2456) [inline app/views/pages/_header.html.erb:41]: I just wanted consistency :smile:

File: db/schema.rb (hunk 2 above)
Comment (aizon2456) [inline db/schema.rb:51]: ProficiencyPosts do not, proficiencies do :+1:

**Pull request merged**